### PR TITLE
Review methodology section

### PR DESCRIPTION
**Pull request by fbarranco**

### CODE REVIEW

File: edubots_surveys.Rnw [hunk 1]
```diff
@@ -826,7 +826,7 @@ Total &  & 193 & 89 & 282 \\ \hline
...
+The majority of the teachers (n=165) mentioned that no changes in their habits occurred due to the emergency remote teaching, as the use of messaging apps was part of their teaching practices and remained the same, from which 105 teachers come from the university and 60 teachers from the non-universitary tertiary sector. The relationships between changes in the use of messaging apps, due to the COVID-19 pandemic, and the instructors' discipline, $X^{2}_{(24, N = 282)} = 44.856$, $p =0.006$, as well as the gender, $X^{2}_{(8, N = 282)} = 16.249$, p= 0.039 were also significant. This finding indicates that how teachers responded to the use of messaging apps during the pandemic was also related to their gender and discipline. In fact, from the majority of the teachers who did not change their habits in this respect (n=165), most of them are males (n=114) and come from the technology (n=60) and engineering (n=53) disciplines (see Tables \ref{tab:gender*covid}, \ref{tab:discipline*covid}).
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
The majority of the teachers (n=165) mentioned that no changes in their habits occurred due to the shift to remote teaching due to the lock-down measures taken in their countries, as the use of messaging apps was part of their teaching practices and remained the same. Out of those 165 responders, 105 teachers come from the university and 60 teachers from the non-universitary tertiary sector. The relationships between changes in the use of messaging apps, due to the COVID-19 pandemic, and the instructors' discipline, $X^{2}_{(24, N = 282)} = 44.856$, $p =0.006$, as well as the gender, $X^{2}_{(8, N = 282)} = 16.249$, p= 0.039 were also significant. This finding indicates that how teachers responded to the use of messaging apps during the pandemic was also related to their gender and discipline. In fact, from the majority of the teachers who did not change their habits in this respect (n=165), most of them are males (n=114) and come from the technology (n=60) and engineering (n=53) disciplines (see Tables \ref{tab:gender*covid}, \ref{tab:discipline*covid}).
```

File: edubots_surveys.Rnw [hunk 2]
```diff
@@ -956,29 +956,29 @@ corrplot(chisq$residuals, is.corr = FALSE)
...
 residuals, showing the difference between the observed and the expected values
 for each cell. Thus, large residuals indicate that variables are not truly
-independent. In our case, the blue shows positive contributions and red negative
-contributions to the test. Moreover, the saturation shows how large the
-contribution is in contrast to the expected value by chance. Additionally,
-tables on the right side show adjusted standardized residuals that according to
+independent. In our case, blue shows positive contributions and red negative
+contributions to the test. Additionally, the saturation shows how large the
+contribution is in contrast to the expected value by chance. 
+Tables on the right side show adjusted standardized residuals that according to
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
Tables show on the right hand side adjusted standardized residuals that according to
```

File: edubots_surveys.Rnw [hunk 3]
```diff
@@ -956,29 +956,29 @@ corrplot(chisq$residuals, is.corr = FALSE)
...
 \cite{agresti2013categorical}, if greater than +/-2 for cases with few cells,
-indicate lack of fit of $H_0$ (in boldface in our table). Complementary, the
+indicates lack of fit of $H_0$ (in boldface in our table). Complementarily, the
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
indicate lack of fit of $H_0$ (in boldface in our table). Complementarily, the
```
this is the predicate for "standardized residuals"

File: edubots_surveys.Rnw [hunk 4]
```diff
@@ -956,29 +956,29 @@ corrplot(chisq$residuals, is.corr = FALSE)
...
 table also shows the percentage of contribution to the test of each cell
-(highest percentage per column also in boldface).
+(the highest percentage per column also in boldface).
 
 Regarding gender in Fig. \ref{figure:gender*postcovid_chisq}, the results show a
 high positive contribution from female educators that did not use a messaging
 app before the COVID-19 pandemic but do use it now; approximately 31\% of the
 test results are explained by this cell. In contrast, male educators that
-respond the same were less than expected (negative contribution that explains
+responded the same were less than expected (negative contribution that explains
```
Comment (JJ) [inline edubots_surveys.Rnw:979]: "responded the same"? The same *answer*? In the same way?

File: edubots_surveys.Rnw [hunk 5]
```diff
@@ -1070,17 +1070,17 @@ corrplot(chisq$residuals, is.corr = FALSE)
     }
     \end{minipage}
 \end{center}
-    \caption{Analysis for discipline and post-covid changes: left) Pearson's residuals for the Chi-square test; right) Discipline*post-covid changes crosstabulation, including standardized residuals \cite{agresti2013categorical} (also adjusted standardized residuals) and contribution percentage to the total Chi-square test of each cell.}
+    \caption{Analysis for discipline and post-COVID-19 changes: left) Pearson's residuals for the Chi-square test; right) Discipline*post-COVID-19 changes crosstabulation, including standardized residuals \cite{agresti2013categorical} (also adjusted standardized residuals) and contribution percentage to the total Chi-square test of each cell.}
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
    \caption{Analysis for discipline and post-COVID-19 changes: left) Pearson's residuals for the Chi-square test; right) Discipline*post-COVID-19 changes cross-tabulation, including standardized residuals \cite{agresti2013categorical} (also adjusted standardized residuals) and contribution percentage to the total Chi-square test of each cell.}
```

**Review by JJ — DISMISSED** [review]
Mostly OK, please take a look at the suggestions.

File: edubots_surveys.Rnw (hunk 4 above)
Comment (fbarranco) [reply edubots_surveys.Rnw:979]: I'm going for "in the same way" in a new commit

File: edubots_surveys.Rnw [hunk 6]
```diff
@@ -466,54 +466,52 @@ We will present next the methodology we followed to find out teachers' opinions.
...
+tertiary or associate degrees that do not have the same consideration as university degrees.}
+To collect this data, two online surveys were designed and developed using Google Forms
+in Spanish. These surveys consisted of six questions on demographic data
+(such as sector, gender, degrees, discipline, age, or teaching experience)
+followed by several multiple-choice questions allowing participants to choose multiple answers.
+
+The {\bf first survey} consisted of multiple-choice questions 
+focused on the use of messaging apps in teaching practice, the type of chatbot use cases that educators would find
+useful for their teaching and the impact of COVID on teaching practice. These questions can be found in their entirity in the Appendix.\footnote{This {\bf first survey} was split into two separate Google forms, one for
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
useful for their teaching and the impact of COVID on teaching practice. These questions can be found in their entirety in the Appendix.\footnote{This {\bf first survey} was split into two separate Google forms, one for
```

File: edubots_surveys.Rnw [hunk 7]
```diff
@@ -466,54 +466,52 @@ We will present next the methodology we followed to find out teachers' opinions.
...
-
-In the {\bf first survey}, the multiple choice questions are
-focused on the use of messaging apps in the teaching practice, the type of chatbot use cases that teachers would consider
-useful for their teaching and the impact of COVID on the teaching practice; please check them out to the full extent in the Appendix.\footnote{This {\bf first survey} was made in two separate Google forms, one for
-university and the other for teachers in tertiary education institutions, outside the university system (vocational training, including 1- and 2-year degrees). Questions and
+This study employed a quantitative approach to address its research objectives. A significant amount of data was collected on the usage of messaging applications and chatbots by educators in universities and colleges.\footnote{In Spain, higher education institutions that would generally known as colleges in other places, are simply called high schools, and provide
+tertiary or associate degrees that do not have the same consideration as university degrees.}
+To collect this data, two online surveys were designed and developed using Google Forms
```
Comment (JJ) [inline edubots_surveys.Rnw]: I wouldn't leave the word "goal" out entirely. Data was gathered to try and achieve one goal.

File: edubots_surveys.Rnw [hunk 8]
```diff
@@ -466,54 +466,52 @@ We will present next the methodology we followed to find out teachers' opinions.
...
+in Spanish. These surveys consisted of six questions on demographic data
+(such as sector, gender, degrees, discipline, age, or teaching experience)
+followed by several multiple-choice questions allowing participants to choose multiple answers.
+
+The {\bf first survey} consisted of multiple-choice questions 
+focused on the use of messaging apps in teaching practice, the type of chatbot use cases that educators would find
+useful for their teaching and the impact of COVID on teaching practice. These questions can be found in their entirity in the Appendix.\footnote{This {\bf first survey} was split into two separate Google forms, one for
+university teachers and another for teachers in tertiary education institutions outside the university system (vocational training, including 1- and 2-year degrees). The questions and
```
Comment (JJ) [inline edubots_surveys.Rnw:479]: maybe specify what the other differed in, instead of simply saying it's "another"?

**Review by JJ — CHANGES_REQUESTED** [review]
Sorry for the additional changes, more stuff came in after my review

File: edubots_surveys.Rnw (hunk 8 above)
Comment (fbarranco) [reply edubots_surveys.Rnw:479]: I think that is explained later. No differences, just targeting university teachers or vocational training teachers

File: edubots_surveys.Rnw (hunk 7 above)
Comment (fbarranco) [reply edubots_surveys.Rnw]: I'm sorry, i don't see it in the excerpt. Just go and change it if you feel like it

File: edubots_surveys.Rnw [hunk 9]
```diff
@@ -466,54 +466,52 @@ We will present next the methodology we followed to find out teachers' opinions.
...
-in Spanish. They are comprised of six questions focusing on demographic data
-(i.e., sector, gender, degrees, discipline, age, teaching experience)
-followed by several multiple choice questions allowing the participants to choose multiple answers.
-
-In the {\bf first survey}, the multiple choice questions are
-focused on the use of messaging apps in the teaching practice, the type of chatbot use cases that teachers would consider
-useful for their teaching and the impact of COVID on the teaching practice; please check them out to the full extent in the Appendix.\footnote{This {\bf first survey} was made in two separate Google forms, one for
-university and the other for teachers in tertiary education institutions, outside the university system (vocational training, including 1- and 2-year degrees). Questions and
+This study employed a quantitative approach to address its research objectives. A significant amount of data was collected on the usage of messaging applications and chatbots by educators in universities and colleges.\footnote{In Spain, higher education institutions that would generally known as colleges in other places, are simply called high schools, and provide
```
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
This study employed a quantitative approach to address its research objectives. In order to fulfill them, we would need to collect a significant amount of data on the usage of messaging applications and chatbots by educators in universities and colleges.\footnote{In Spain, higher education institutions that would generally known as colleges in other places, are simply called high schools, and provide
```

File: edubots_surveys.Rnw (hunk 7 above)
Comment (JJ) [inline edubots_surveys.Rnw]:
```suggestion
In order to collect this data, two online surveys were designed and developed using Google Forms
```

**Review by JJ — APPROVED** [review]
Finally rephrased slightly the paragraph. Please take a look at the suggestions; it's good other than that.

**Pull request merged**